{"uuid":"293f342e-a7a9-47e0-81f8-da0b395ca19a","historyId":"36030a09f9b1923da241513131a6c206","fullName":"tests.ErrorValidationTest.testInvalidLoginLandingPage","labels":[{"name":"package","value":"tests.ErrorValidationTest"},{"name":"testClass","value":"tests.ErrorValidationTest"},{"name":"testMethod","value":"testInvalidLoginLandingPage"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"tests.ErrorValidationTest"},{"name":"host","value":"Qcentris-396"},{"name":"thread","value":"408@Qcentris-396.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Validates error message for invalid login attempts on the landing page.","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Cannot invoke \"String.getBytes(java.nio.charset.Charset)\" because \"content\" is null","trace":"java.lang.NullPointerException: Cannot invoke \"String.getBytes(java.nio.charset.Charset)\" because \"content\" is null\r\n\tat io.qameta.allure.Allure.addAttachment(Allure.java:439)\r\n\tat tests.ErrorValidationTest.testInvalidLoginLandingPage(ErrorValidationTest.java:64)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\n"},"stage":"finished","description":"This test performs an invalid login and validates the error message displayed on the landing page.","steps":[],"attachments":[{"name":"Test Body","source":"4c335b62-ebf5-47f3-b0ea-67d687553166-attachment.txt","type":"text/plain"},{"name":"InvalidLoginErrorScreenshot","source":"723e1aeb-a012-4ea7-ab94-a5e9878a404b-attachment"}],"parameters":[],"start":1736068142578,"stop":1736068142870}